cypress/support/config/settings.js

```diff
@@ -8183,233 +8183,7 @@ module.exports = () => ({
       },
     },
   },
-  uzbek: {
```

Comment:
Oh! I wasn't expecting this, only the removal of the frontPage code. 

It might be difficult to determine whether this is going to cause an issue in the pipeline/scheduled E2Es just by looking at this code. I wouldn't want us to have to roll back the entire release just because the tests are not quite right. 

Can we just delete the frontPage test section and leave the rest. In a future PR, after the release, we can tidy up this and other uzbek stuff.

Reply:
Deleting this means we're missing coverage for all other non-variant pages - articles, live radio etc, so we definitely need to keep it for now.

Reply:
Do you mean then to make sure all the uzbek test settings are like this? 
`frontPage: { environments: undefined, smoke: false }, ` (they are all like this already)

Or do you mean to delete that line entirely. I thought doing that caused errors before but I can try it again if that is what you mean.

Reply:
Why is ther 'uzbek' as well as 'uzbekCyr' and 'uzbekLat'? 🤔 The other variant services don't have that version.

Reply:
I have put this back now 👍